src/Features/CSharp/Portable/SignatureHelp/SignatureHelpUtilities.cs

```diff
@@ -15,23 +15,29 @@ internal static class SignatureHelpUtilities
     {
         private static readonly Func<BaseArgumentListSyntax, SyntaxToken> s_getBaseArgumentListOpenToken = list => list.GetOpenToken();
         private static readonly Func<TypeArgumentListSyntax, SyntaxToken> s_getTypeArgumentListOpenToken = list => list.LessThanToken;
+        private static readonly Func<InitializerExpressionSyntax, SyntaxToken> s_getInitializerExpressionOpenToken = e => e.OpenBraceToken;
```

Comment:
note we could use generics to greatly simplify this across all sighelpproviders.  but i've avoided that for now to keep things simpler.  i may do that in a PR in the future.